docs/application/toc_all.md

```diff
@@ -232,12 +219,37 @@
 #### [TEE Communication](/application/dotnet/guides/security/tee-client.md)
 #### [Device Policy Management](/application/dotnet/guides/security/dpm.md)
 
-### [Natural Language Processing](/application/dotnet/guides/nlp/nlp.md)
-
+### Sensors
+#### [Overview](/application/dotnet/guides/sensors/overview.md)
+#### [Device Sensors](/application/dotnet/guides/sensors/device-sensors.md)
+
+### System
+#### [Overview](/application/dotnet/guides/system/overview.md)
+#### [Attached Devices](/application/dotnet/guides/system/attached-devices.md)
+#### [System Information](/application/dotnet/guides/system/system.md)
+#### [System Settings](/application/dotnet/guides/system/settings.md)
+#### [Sound and Vibration Feedback](/application/dotnet/guides/system/feedback.md)
+
+###  Telephony
+#### [Overview](/application/dotnet/guides/telephony/index.md)
+#### [Telephony Information](/application/dotnet/guides/telephony/telephony.md)
+#### [Phone Number Management](/application/dotnet/guides/telephony/phonenumber.md)
+
+### UIX
+#### [Overview](/application/dotnet/guides/uix/overview.md)
+#### [Input Method](/application/dotnet/guides/uix/input-method.md)
+#### [Input Method Manager](/application/dotnet/guides/uix/input-method-manager.md)
+#### [Speech-to-text](/application/dotnet/guides/uix/stt.md)
+#### [Text-to-speech](/application/dotnet/guides/uix/tts.md)
+#### [Voice Control](/application/dotnet/guides/uix/voice-control.md)
+#### [Voice Control Manager](/application/dotnet/guides/uix/voice-control-manager.md)
+
+### WebView
+#### [Overview](/application/dotnet/guides/webview/overview.md)
```

Comment:
The title for both   #### [Overview](/application/dotnet/guides/webview/overview.md) and #### [WebView](/application/dotnet/guides/webview/webview.md are same, request to differentiate the page level headings.

Reply:
Overview page title has been updated.

Reply:
Inconsistency in naming: 

The overview pages have heading same as the topic heading in toc , here in this case it differs: 
https://docs1.stg.tizen.org/staging/1387/application/dotnet/guides/webview/overview/

WebView Text in ToC must be changed to the heading in Overview for consistency. 

- Please change WebView in Toc to Web Content Displaying or (Display Web Content)

![image](https://user-images.githubusercontent.com/43839737/115208278-0335a100-a11a-11eb-95a7-b87c9c8a163f.png)


Toc heading must be same as page heading of overview.

I am suggesting to use 
Display Web Content as heading instead of Web Content Displaying

Reply:
The toc heading has been updated to 'Display Web Content'.